LanguageWeaverProvider/XliffConverter/SegmentParser/Parser.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Sdl.LanguagePlatform.Core;
+
+namespace LanguageWeaverProvider.XliffConverter.SegmentParser
+{
+	public class Parser
```

Comment:
I know this is older code, but are you certain that this parser is sufficiently general, or do you have insurance from Language Weaver that their the xliffs will not change in a way that breaks this regex based parser?
I think an example of a xml/xliff-conformant change that would break this parser would be to add a space in stand-alone tags as so: '<1 id=1 text-equiv="bla" />' instead of '"<1 id=1 text-equiv="bla"/>"', or to switch from " to ' quotes.

Reply:
PS: I am realizing that what is parsed here looks a lot like XML, but is not actual xml (!) and is not the xliff itself, but rather some xml-alike string embedded in the xliff. I am confused, but maybe this means its all fine and regex-based parsing is the only option.